google/cloud/forseti/services/cli.py

```diff
@@ -1006,7 +1020,10 @@ def main(args,
     if not services:
         services = SERVICES
     output = outputs[config.out_format]()
-    services[config.service](client, config, output, config_env)
+    try:
+        services[config.service](client, config, output, config_env)
+    except ValueError as e:
+        parser.error(e.message)
```

Comment:
Do you think it's better to centralize the error checking logic inside the parser? So instead of having the check in each of the explain methods, we only need to check them to the parser.

Reply:
This is the case already apart from the 2 notable exceptions I am fixing in this PR.
`argparse` does not support the following semantics: "I want at least one of the following args/options"
That's why such a case needs to be checked manually.